plasmapy/particles/decorators.py

```diff
@@ -43,7 +57,7 @@
 )
 
 
-def _get_annotations(callable_: Callable):
+def _get_annotations(callable_: Callable[..., Any]) -> dict[str, Any]:
```

Comment:
For the `Callable[..., Any]` annotation, the `...` represents that the callable can accept whatever arguments, and the `Any` represents that the function can have any return type.